src/shapes/polyline.class.ts

```diff
@@ -154,28 +152,34 @@ import { degreesToRadians } from '../util/misc/radiansDegreesConversion';
             pathOffset: new Point(),
           };
         }
-        const bbox = makeBoundingBoxFromPoints(points);
-        const bboxNoStroke = makeBoundingBoxFromPoints(this.points);
-        const offsetX = bbox.left + bbox.width / 2,
-          offsetY = bbox.top + bbox.height / 2;
-        const pathOffsetX =
-          offsetX - offsetY * Math.tan(degreesToRadians(this.skewX));
-        const pathOffsetY =
-          offsetY - pathOffsetX * Math.tan(degreesToRadians(this.skewY));
-        // TODO: remove next line
-        const legacyCorrection =
-          !this.fromSVG && !this.exactBoundingBox ? this.strokeWidth / 2 : 0;
+        const bbox = makeBoundingBoxFromPoints(points),
+          // Remove scale effect, since it's applied after
+          matrix = calcDimensionsMatrix(
+            Object.assign(options, { scaleX: 1, scaleY: 1 })
```

Comment:
Is this intentional?
It mutates options
```suggestion
            { ...options, scaleX: 1, scaleY: 1 }
```

Reply:
no, I made a mistake. Thanks!